src/main/java/codeu/model/store/basic/UserStore.java

```diff
@@ -66,11 +66,21 @@ private UserStore(PersistentStorageAgent persistentStorageAgent) {
     users = new ArrayList<>();
 
     // hard-coded initial Admin:
-    // TODO(JW): Merge "the creation process" from RegisterServlet to here.
-    String hashedPassword = BCrypt.hashpw("AdminPass01", BCrypt.gensalt());
-    User initialAdmin = new User(UUID.randomUUID(), "Admin01", hashedPassword, Instant.now());
-    initialAdmin.setAdmin(true);
-    users.add(initialAdmin);
+    this.addUser("Admin01", "AdminPass01", true);
```

Comment:
Since it won't be clear to the reader what the `true` refers to without them having to find the method signature, it's aften helpful to add a little comment with the name of the parameter like this: `this.addUser("Admin01", "AdminPass01", /*adminStatus=*/true);`

Reply:
I see, fixed!